library/src/main/java/com/airbnb/android/airmapview/AirMapView.java

```diff
@@ -79,7 +82,7 @@ public void initialize(FragmentManager fragmentManager, AirMapInterface mapInter
    *
    * @param fragmentManager required for initialization
    */
-  public void initialize(FragmentManager fragmentManager) {
+  public void initialize(FragmentManager fragmentManager, String mapboxAccessToken, String mapboxMapId) {
```

Comment:
So as to support older versions, we should overload `initialize` so that we don't break users of the old API